code/drasil-code/Language/Drasil/Chunk/Code.hs

```diff
@@ -128,16 +128,17 @@ data CodeChunk = CodeC { _qc :: QuantityDict
                        }
 makeLenses ''CodeChunk
 
-instance HasUID    CodeChunk where uid = qc . uid
-instance NamedIdea CodeChunk where term = qc . term
-instance Idea      CodeChunk where getA = getA . view qc
-instance HasSpace  CodeChunk where typ = qc . typ
-instance HasSymbol CodeChunk where symbol c = symbol (c ^. qc)
-instance Quantity  CodeChunk where getUnit = getUnit . view qc
-instance CodeIdea  CodeChunk where
+instance HasUID      CodeChunk where uid = qc . uid
+instance NamedIdea   CodeChunk where term = qc . term
+instance Idea        CodeChunk where getA = getA . view qc
+instance HasSpace    CodeChunk where typ = qc . typ
+instance HasSymbol   CodeChunk where symbol c = symbol (c ^. qc)
+instance Quantity    CodeChunk where 
+instance CodeIdea    CodeChunk where
   codeName (CodeC c Var) = symbToCodeName (codeSymb c)
   codeName (CodeC c Func) = funcPrefix ++ symbToCodeName (codeSymb c)
-instance Eq        CodeChunk where c1 == c2 = (c1 ^. uid) == (c2 ^. uid)
+instance Eq          CodeChunk where c1 == c2 = (c1 ^. uid) == (c2 ^. uid)
+instance MayHaveUnit CodeChunk where getUnit u = getUnit $ u ^. qc
```

Comment:
Could you please code these in the same style as they were done before?  For example
```haskell
instance MayHaveUnit  CodeChunk where getUnit = getUnit . view qc
```
(which I got by cut-and-paste from the old `Quantity` instance.  All throughout, I would prefer if you used the coding style that was there in the `Quantity` instances for the lens, rather than the more 'pointwise' style you have used now.